src/mudlet.cpp

```diff
@@ -2976,6 +2977,12 @@ void mudlet::slot_multi_view(const bool state)
     }
 }
 
+void mudlet::slot_caret_mode(bool state)
+{
+    // FIXME: Support multiple hosts.
```

Comment:
Yeah, you will need to handle multiple `host` instances - and using `// FIXME:` is not making this PR "dangerous" (upsetting the *Danger* Bot that watches some aspects of PRs!) whereas using `// TODO:` definitely would... 😲

Reply:
I plan to implement this before making the definitive PR. This comment is just so that I don’t forget about it. :)